vcr/assets/NutsOrganizationCredential.config.yaml

```diff
@@ -0,0 +1,34 @@
+concept: organization
```

Comment:
should we add the license comment block here as well? (same for `NutsAuthorizationCredential.config.yaml`)

Reply:
no intellectual property to protect is there?